sonar-plugin-api/src/main/java/org/sonar/api/profiles/ProfileExporter.java

```diff
@@ -32,6 +33,7 @@
  */
 @BatchSide
 @ServerSide
+@ComputeEngineSide
```

Comment:
Initially added it because it's in the API.

I don't know that part, so I'll blindly trust you on this and will remove it